README.md

```diff
@@ -23,13 +25,13 @@ A simple setup using:
 1. Curl the `setup.sh` script and run it with bash on your machine:
 
 ```bash
-curl -fsSL https://raw.githubusercontent.com/kensonjohnson/relativepath.tech/chris/npm-ci/setup.sh | bash -
+curl -fsSL https://raw.githubusercontent.com/kensonjohnson/relativepath.tech/main/setup.sh | bash -
```

Comment:
We shouldn't be using a second curl here. If the above script runs, it clones the repo and they have access to the `test.sh` locally. Also, there is nothing to test until _after_ they run `npm run dev`